### PR TITLE
Change Restaurant WebClip from custom element to set of html tags

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/1392
[Problem] Error when adding Restaurant webclip again
[Solution] Change Restaurant WebClip form custom element to set of html tags.
Move additional pages to webclip Restaurant folder.

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: examples/mobile/HomeApp/webclip/restaurant/js/app.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var sectionChanger = document.querySelector(".ui-section-changer"),
+	tau = window.tau;
+
+tau.engine.createWidgets(sectionChanger);
+
+// refresh section changer
+window.setTimeout(function () {
+	tau.engine.getBinding(sectionChanger).refresh();
+}, 200);
```
Comment (DohyungLim) [inline examples/mobile/HomeApp/webclip/restaurant/js/app.js:9]: Is there any effect of timeout?
Are the results different over time?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

**Review by singa2000 — APPROVED** [review]
LGTM.
Thanks

File: examples/mobile/HomeApp/webclip/restaurant/js/app.js (hunk 1 above)
Comment (lslachciak) [reply examples/mobile/HomeApp/webclip/restaurant/js/app.js:9]: this code comes from original webclip.
When no timeout or timeout=0 below error occurs and webclip is not visible:
```
tau][9/7/2020, 8:59:25 AM] TypeError: Cannot read property 'refresh' of undefined
    at eval (eval at <anonymous> (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:37788:14), <anonymous>:4:41)
    at eval (<anonymous>)
    at Array.<anonymous> (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:37788:14)
    at Object.batchCall (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:37814:22)
    at Object.util.importEvaluateAndAppendElement (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:37912:10)
    at Card.prototype._include (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:28764:24)
    at Card.prototype.changeContent (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:28772:20)
    at CardRoute.prototype.open (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:60795:11)
    at Object.resolve (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:15734:13)
    at Router._loadSuccess (http://106.120.63.110:3000/lib/tau/mobile/js/tau.js:16408:15)
ns.error @ tau.js:37468
(anonymous) @ tau.js:37791
batchCall @ tau.js:37814
util.importEvaluateAndAppendElement @ tau.js:37912
prototype._include @ tau.js:28764
prototype.changeContent @ tau.js:28772
prototype.open @ tau.js:60795
resolve @ tau.js:15734
Router._loadSuccess @ tau.js:16408
(anonymous) @ tau.js:16354
targetCallback @ tau.js:15270
templateCallback @ tau.js:15275
callbackFunction @ tau.js:60847
load (async)
htmlTemplate @ tau.js:60875
render @ tau.js:15292
Router._loadUrl @ tau.js:16351
Router._openDeferred @ tau.js:15743
Router.open @ tau.js:15803
prototype._init @ tau.js:28743
prototype.init @ tau.js:5827
coreProcessWidget @ tau.js:3353
processWidget @ tau.js:3402
processHollowWidget @ tau.js:3519
processBuildQueueItem @ tau.js:3556
createWidgets @ tau.js:3622
updateWebClipsUI @ app.js:98
(anonymous) @ app.js:43
Promise.finally (async)
loadWebClipList @ app.js:42
init @ app.js:170
onPageBeforeShow @ app.js:175
trigger @ tau.js:1636
prototype.trigger @ tau.js:6453
prototype.onBeforeShow @ tau.js:14875
prototype.change @ tau.js:12125
routePage.open @ tau.js:59631
resolve @ tau.js:15734
Router._openDeferred @ tau.js:15748
Router.open @ tau.js:15803
Router.register @ tau.js:16143
Router.init @ tau.js:15919
(anonymous) @ tau.js:16584
trigger @ tau.js:1636
build @ tau.js:3686
(anonymous) @ tau.js:1960

```

**Comment by lslachciak** [comment]
I created separate issue to improve section changer refresh https://github.com/Samsung/TAU/issues/1395

**Pull request merged**